message FeatureItem {
    required string field_name = 1;
    repeated int64 value = 2;
}

message User {
    required string user_id = 1;
    repeated FeatureItem feture_item = 2;
}

message UserList {
    repeated User user = 1;
}